src/common/tests/CommonUtilsUT.cpp

```diff
@@ -2062,6 +2062,16 @@ TEST_F(CommonUtilsTest, CheckInstallUninstallPackage)
     EXPECT_NE(0, CheckPackageNotInstalled("gcc", nullptr, nullptr));
 }
 
+TEST_F(CommonUtilsTest, CheckPackageManagerNotThrottling)
+{
+    for (int i = 0; i < 10; i++)
```

Comment:
Is 10 sufficient?

Reply:
It's 10 x 3, each line executed as new command. Would you suggest more?

Reply:
Increased to 100, please review